iroha_p2p/src/lib.rs

```diff
@@ -34,3 +40,23 @@ pub enum Error {
 
 /// Result to use in this crate.
 pub type Result<T, E = Error> = std::result::Result<T, E>;
+
+/// Any message read from other peer, serialized to bytes
+#[derive(Debug, Clone, Encode, Decode, iroha_actor::Message)]
+pub struct Message(pub Vec<u8>);
+
+/// Read result from any peer
+#[derive(Debug, iroha_actor::Message)]
+pub struct MessageResult(pub Result<Message, Error>);
+
+impl MessageResult {
+    /// Convenience constructor for positive result
+    pub const fn new_message(message: Message) -> Self {
+        Self(Ok(message))
+    }
+
+    /// Convenience constructor for negative result
+    pub const fn new_error(error: Error) -> Self {
+        Self(Err(error))
+    }
+}
```

Comment:
Don't see places where these constructors are used. Can be deleted?

Reply:
`peer.rs:133` for example.